homeassistant/components/tuya/number.py

```diff
@@ -84,6 +84,36 @@
             icon="mdi:microphone",
         ),
     ),
+    # Motion Sensor Light
+    # https://developer.tuya.com/en/docs/iot/gyd?id=Kaof8a8hycfmy
+    "gyd": (
+        NumberEntityDescription(
+            key=DPCode.PIR_DELAY,
+            name="PIR Delay",
+            native_unit_of_measurement="Secs",
+            icon="mdi:timer-play",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        NumberEntityDescription(
+            key=DPCode.STANDBY_TIME,
+            name="Standby Time",
+            native_unit_of_measurement="Mins",
+            icon="mdi:timer-plus",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        NumberEntityDescription(
+            key=DPCode.STANDBY_BRIGHT,
+            name="Standby Bright",
+            icon="mdi:brightness-6",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        NumberEntityDescription(
+            key=DPCode.PIR_RESUME_COUNTDOWN,
+            name="Resume Countdown",
+            icon="mdi:av-timer",
+            entity_category=EntityCategory.CONFIG,
+        ),
```

Comment:
Does this count down actively? As in, you do see the number going down while the countdown happens?

Reply:
The number counts down every minute if the PIR sensor is "switched off" when the device is in Auto mode. At 0 the PIR sensor is turned back on. The Tuya documentation suggests that this is a attribute that can be set, but in the case of the floodlight I was testing with, this field seems to always start at 5 minutes - so possible should be exposed to HA as a "sensor" rather than "number input"

Reply:
> The number counts down every minute if the PIR sensor is "switched off" when the device is in Auto mode.

A number is for settings and should not be used for countdowns.
You can use a countdown sensor (sensor with a targeted date the countdown would be finished) and a number sensor for setting the base countdown.

The current construct isn't allowed.

Reply:
I have removed this attribute and it doesn't appear to add any useful function to the integration.